.openpublishing.redirection.json

```diff
@@ -1178,6 +1178,11 @@
             "source_path": "aspnetcore/blazor/forms-validation.md",
             "redirect_url": "/aspnet/core/blazor/forms-and-input-components",
             "redirect_document_id": false
+        },
+        {
+            "source_path": "aspnetcore/core/grpc/json-transcoding.md",
+            "redirect_url": "/aspnet/core/grpc/httpapi",
+            "redirect_document_id": false
```

Comment:
I don't know what to put here for a redirect to work. I want `/aspnet/core/grpc/httpapi` to be redirected to `/aspnet/core/grpc/json-transcoding`